<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>130</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>130</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>130</width>
    <height>130</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>130</width>
    <height>130</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centalwidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <widget class="RadioGroup" name="widgetMain" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonSelectImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>装载单个镜像</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMountPart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>装载整分区</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMountSerial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>连接串口</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMountNic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>模拟网卡</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonChangePassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonClean">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
   <widget class="WidgetMountBlock" name="widgetMountBlock" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <widget class="QListWidget" name="listWidgetBlockInfo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetMountSerial" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <widget class="QLabel" name="labelSerialInfo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>串口连接成功</string>
     </property>
    </widget>
   </widget>
   <widget class="WidgetSelectImage" name="widgetSelectImage" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <widget class="QListWidget" name="listWidgetImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>130</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetMountNic" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <widget class="QLabel" name="labelNicInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>网卡连接成功</string>
     </property>
    </widget>
   </widget>
   <widget class="WidgetGetPassword" name="widgetGetPass" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <widget class="PasswordEdit" name="lineEditPassword" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输入密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>42</y>
       <width>110</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>回退: 方向左
确认: Press
K1: 上一字符
K2: 右移
K3:下一字符</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetWait" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <widget class="QLabel" name="labelWaitInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>请等待</string>
     </property>
    </widget>
   </widget>
   <widget class="WidgetChangePassword" name="widgetChangePassword" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <widget class="PasswordEdit" name="lineEditOldPassword" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="PasswordEdit" name="lineEditNewPassword" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="PasswordEdit" name="lineEditNewPasswordAgain" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelOldPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>老密</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>新密</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewPasswordAgain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
   <widget class="WidgetResult" name="widgetResult" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelResultTitle">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelResultInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="WidgetSelectNic" name="widgetSelectNic" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <widget class="QTreeWidget" name="treeWidgetNic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>130</height>
      </rect>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>60</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RadioGroup</class>
   <extends>QWidget</extends>
   <header>radiogroup.h</header>
  </customwidget>
  <customwidget>
   <class>PasswordEdit</class>
   <extends>QWidget</extends>
   <header>passwordedit.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetSelectImage</class>
   <extends>QWidget</extends>
   <header>widgetselectimage.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetChangePassword</class>
   <extends>QWidget</extends>
   <header>widgetchangepassword.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetResult</class>
   <extends>QWidget</extends>
   <header>widgetresult.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetMountBlock</class>
   <extends>QWidget</extends>
   <header>widgetmountblock.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetSelectNic</class>
   <extends>QWidget</extends>
   <header>widgetselectnic.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetGetPassword</class>
   <extends>QWidget</extends>
   <header>widgetgetpassword.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
